packages/react/xr/src/origin.tsx

```diff
@@ -4,29 +4,38 @@ import { Group } from 'three'
 import { xrSpaceContext } from './contexts.js'
 import { useXR } from './xr.js'
 
+export type XROriginProps = ThreeElements['group'] & {
+  disabled?: boolean
+}
+
 /**
  * Component for setting the origin of the player (their feet)
  *
  * @param props
  * Accepts the same props as a ThreeJs [Group](https://threejs.org/docs/#api/en/objects/Group)
  * @function
  */
-export const XROrigin = forwardRef<Group, ThreeElements['group']>(({ children, ...props }, ref) => {
+export const XROrigin = forwardRef<Group, XROriginProps>(({ children, disabled, ...props }, ref) => {
   const xrCamera = useThree((s) => s.gl.xr.getCamera())
   const internalRef = useRef<Group>(null)
-  useImperativeHandle(ref, () => internalRef.current!, [])
   const referenceSpace = useXR((xr) => xr.originReferenceSpace)
+
+  useImperativeHandle(ref, () => internalRef.current!, [])
+
   useEffect(() => {
     const group = internalRef.current
-    if (group == null) {
+    if (!group || disabled) {
       return
     }
+
     group.add(xrCamera)
+
     return () => void group.remove(xrCamera)
-  }, [xrCamera])
+  }, [disabled, xrCamera])
+
   return (
     <group ref={internalRef} {...props}>
-      {referenceSpace != null && <xrSpaceContext.Provider value={referenceSpace}>{children}</xrSpaceContext.Provider>}
+      {referenceSpace && <xrSpaceContext.Provider value={referenceSpace}>{children}</xrSpaceContext.Provider>}
```

Comment:
Should this still be rendered if it's disabled? Also what are some use cases for rendering something inside an XROrigin? 

I had to work around children not being rendered when a reference space is undefined (e.g. when not in XR inside Triplex) to render its helper.

Reply:
Okay! Will do